### PR TITLE
Admin revamp

### PR DESCRIPTION
**Pull request by lunika**
admin is now managed by zoapp-front and I kept in opla-front only panels related to bots management

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "request-promise-native": "^1.0.5",
     "timezones.json": "^1.3.3",
     "zoapp-common": "0.3.0",
-    "zoapp-front": "0.18.5",
+    "zoapp-front": "0.19.0",
```
Comment (mikbry) [inline package.json]: I have some trouble with Advanced and null emailServer
screen shot 2018-05-25 at 12 16 04
Rechecked several time that opla-backend is up to date.

Comment (lunika) [reply package.json]: I don't think this is a opla-backend issue but I also don't know how `emailServer` can be null...
I will manage this case in zoapp-front

Comment (lunika) [reply package.json]: can you test with this version of zoapp-front ? https://github.com/Zoapp/front/pull/66/files

Comment (mikbry) [reply package.json]: Same error with zoapp-front@0.19.0 , I checked in node_modules version is 0.19, but the code seems the same as previous.

Comment (lunika) [reply package.json]: hmm the code is not the same so I think you don't have the code from the PR linked in my previous comment

Comment (mikbry) [reply package.json]: Delete node_modules and install back 0.19. Same the advanced.js code is same as previous

Comment (lunika) [reply package.json]: No I never told you to install back 0.19, but the code from the PR I linked above. There is no release of it and it doesn't concern the backend but only` zoapp-front`, this one https://github.com/Zoapp/front/pull/66/files

Comment (mikbry) [reply package.json]: Aaah got it
But there is a little bug ;-)
screen shot 2018-05-25 at 17 29 11

**Pull request merged**